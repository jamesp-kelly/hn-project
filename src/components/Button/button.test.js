import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Button from './index';

describe('Button', () => {

  const props = {
    onClick: () => {}
  };

  it('renders', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Button { ...props}>More</Button>, div);
  });

  test('snapshots', () => {
    const component = renderer.create(<Button { ...props}>More</Button>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});