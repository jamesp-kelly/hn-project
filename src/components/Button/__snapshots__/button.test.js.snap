exports[`Button snapshots 1`] = `
<button
  className=""
  onClick={[Function]}
  type="button">
  More
</button>
`;
